app/assets/stylesheets/forever_style_guide/modules/_checkbox.scss

```diff
@@ -1,69 +1,68 @@
-// Styling checkboxes and radio buttons by hiding the input and using pseudo-classes
-// This technique is a bit more tightly coupled to the markup than we'd prefer but it gives us a lot of cross-browser style control
-.checkbox.checkbox-branded,
-.checkbox-inline.checkbox-branded,
-.radio.radio-branded,
-.radio-inline.radio-branded {
-  input[type="checkbox"],
-  input[type="radio"] {
-    opacity: 0;
-    width: 0;
-    height: 0;
-  }
-  .checkbox-visual,
-  .radio-visual {
-    margin-right: 15px;
-    float: left;
+$checkbox_size: 20px;
 
-    &:before {
-      @extend %check-style;
-      background-color: color('gray-200');
-      color: color('gray-400');
-      font-family: fontawesome;
-    }
-  }
-  input[type="checkbox"]:checked + .checkbox-visual:before,
-  input[type="radio"]:checked + .radio-visual:before {
-    color: color('white');
-    background-color: color('primary');
-  }
-  .checkbox-label,
-  .radio-label {
-    display: block;
-    padding-left: 0;
-    line-height: $btn-xs-size;
-  }
+@mixin checkbox($background-color, $border-color) {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: $checkbox_size;
+  height: $checkbox_size;
+  background-color: $background-color;
+  border: 2px solid $border-color;
+  border-radius: $border-radius-large;
+  @include transition(border-color 0.2s ease-in-out, background-color 0.2s ease-in-out);
 }
 
-.checkbox-inline.checkbox-branded,
-.radio-inline.radio-branded {
-  padding-left: 0;
+.checkbox-branded {
+  position: relative;
+  font-size: $font-size-default;
 
-  .checkbox-label,
-  .radio-label {
+  &.checkbox { // using added specificity of .checkbox to override bootstrap
     display: inline-block;
-    margin-right: 25px;
+ 
+    label {
+      padding-left: $checkbox_size + 10px;
```

Comment:
line 67 below also sums to 30px but does so by *1.5. Shall we pick a way to be consistent?